### PR TITLE
KT navigation redesign

### PR DESCRIPTION
**Pull request by hartantothio**

### CODE REVIEW

**Comment by bbejeck** [comment]
@hartantothio can you rebase this PR?

File: _includes/nav.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
   }
 </style>
 
-<header class="header" data-swiftype-index="false">
```
Comment (scott-confluent) [inline _includes/nav.html:12]: Is there a reason you're removing this?

File: _includes/nav.html [hunk 2]
```diff
@@ -146,89 +109,34 @@
...
-                ksqlDB</a></li>
-            <li><a href="https://developer.confluent.io/learn-kafka/spring/confluent-cloud/">Spring Framework and Apache
-                Kafka</a></li>
-            <li><a href="https://developer.confluent.io/learn-kafka/data-pipelines/intro/">Building Data Pipelines with
-                Apache Kafka and Confluent</a></li>
-            <li><a href="https://developer.confluent.io/learn-kafka/event-sourcing/event-driven-vs-state-based/">Event
-                Sourcing and Event Storage with Apache Kafka</a></li>
-            <li><a href="https://developer.confluent.io/learn-kafka/architecture/get-started/">Kafka Internals</a></li>
-          </ul>
+          <a rel="noopener noreferrer" class="py50" href="https://developer.confluent.io/learn-kafka/">Courses</a>
```
Comment (scott-confluent) [inline _includes/nav.html:112]: What is `py50`?

**Comment by scott-confluent** [comment]
I don't see [icon-book-circled.png](https://github.com/confluentinc/kafka-tutorials/pull/1362/files#diff-b848695b19573917c1ef51bbfebc9af49687c5b18e64f92ddb49e005adda1731) being used. Also, it's gigantic. Do we need it here?

File: _includes/nav.html (hunk 2 above)
Comment (bpurkaystha) [reply _includes/nav.html:112]: `padding: 50px 0`

File: _includes/nav.html (hunk 1 above)
Comment (bpurkaystha) [reply _includes/nav.html]: I didn't intend to remove this. Probably, removed due to merge conflict resolution. Hence, brought back

File: assets/sass/_nav.scss [hunk 3]
```diff
@@ -643,3 +690,374 @@ $font-family-secondary: "MarkPro-Medium", Arial, sans-serif;
...
+  @mixin nav-item-group-wrapper-commons {
+    max-width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 30px;
+  }
+
+  @include for-max-mobile {
+
```
Comment (scott-confluent) [inline assets/sass/_nav.scss:1051]: nit: remove extra line

File: assets/sass/_nav.scss [hunk 4]
```diff
@@ -643,3 +690,374 @@ $font-family-secondary: "MarkPro-Medium", Arial, sans-serif;
...
+    @include nav-item-group-wrapper-commons
+  }
+
+  @include for-max-tablet {
+
```
Comment (scott-confluent) [inline assets/sass/_nav.scss:1056]: nit: remove extra line

File: assets/sass/_nav.scss [hunk 5]
```diff
@@ -643,3 +690,374 @@ $font-family-secondary: "MarkPro-Medium", Arial, sans-serif;
...
+        max-width: 100%;
+      }
+
+      @include for-max-mobile {
+        max-width: 100%;
+      }
+    }
+
+    .groupWrapperHeading, .navItemGroupWrapper {
+
```
Comment (scott-confluent) [inline assets/sass/_nav.scss:719]: nit: remove extra line

File: assets/sass/_nav.scss [hunk 6]
```diff
@@ -182,40 +195,34 @@ $font-family-secondary: "MarkPro-Medium", Arial, sans-serif;
...
+                font-size: 14px;
                 font-weight: 300;
                 width: 100%;
-                display: block;
-                white-space: nowrap;
                 overflow: hidden;
                 text-overflow: ellipsis;
                 padding: 10px 0;
-                margin: 0 10px;
+                //margin: 0 10px;
```
Comment (scott-confluent) [inline assets/sass/_nav.scss:215]: remove

File: assets/sass/_nav.scss [hunk 7]
```diff
@@ -146,7 +159,7 @@ $font-family-secondary: "MarkPro-Medium", Arial, sans-serif;
           }
 
           > a {
-            display: block;
+            //display: block;
```
Comment (scott-confluent) [inline assets/sass/_nav.scss:162]: remove

File: _includes/navMenus/learn.html [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+                    <img src="/tutorials/assets/img/icon-function-light.svg" alt="" width="25" height="25"/>
+                    <div>
+                        <div class="title">100 Days of code</div>
+                        <span class="navItemSubline">A self-directed learning path</span>
+                    </div>
+                </a>
+            </li>
+        </ul>
+        <ul>
+        <li class="navItem">
```
Comment (scott-confluent) [inline _includes/navMenus/learn.html:52]: fix indentation

File: _includes/navMenus/learn.html [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+            <div class="title">Pick your learning path</div>
+            <span class="subline">A wide range of resources to get you started</span>
+            <a href="https://developer.confluent.io/learn-kafka/" class="navItemGroupCta">
+                Start Learning
+                <img src="/tutorials/assets/img/icon-arrow-right-white.svg" width="15" height="15"/>
+            </a>
+        </div>
+    </ul>
+    <ul class="navItemMainColumns">
+        <ul>
```
Comment (scott-confluent) [inline _includes/navMenus/learn.html:13]: not sure you can nest a `ul` within a `ul` directly

File: _includes/navMenus/docs.html [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+                Use the Cloud quick start to get up and running with Confluent Cloud using a basic cluster
+            </span>
+            <a href="https://docs.confluent.io/cloud/current/get-started/index.html" class="navItemGroupCta">
+                Learn more
+                <img src="/tutorials/assets/img/icon-arrow-right-white.svg" width="15" height="15"/>
+            </a>
+        </div>
+    </ul>
+    <ul class="navItemMainColumns">
+        <ul>
```
Comment (scott-confluent) [inline _includes/navMenus/docs.html:15]: not sure you can nest a `ul` within a `ul` directly

File: _includes/navMenus/docs.html [hunk 11]
```diff
@@ -0,0 +1,74 @@
+<ul class="menuGroup docs">
+    <ul class="wrapperSection">
```
Comment (scott-confluent) [inline _includes/navMenus/docs.html:2]: not sure you can nest a 'ul' within a 'ul' directly

File: _includes/navMenus/learn.html [hunk 12]
```diff
@@ -0,0 +1,90 @@
+<ul class="menuGroup learn">
+    <ul class="wrapperSection">
```
Comment (scott-confluent) [inline _includes/navMenus/learn.html:2]: not sure you can nest a 'ul' within a 'ul' directly

File: _includes/navMenus/courses.html [hunk 13]
```diff
@@ -0,0 +1,94 @@
...
+                        View all courses
+                        <img src="/tutorials/assets/img/icon-arrow-right-white.svg" width="15" height="15"/>
+                    </a>
+                </div>
+            </div>
+        </div>
+    </ul>
+
+    <ul class="navItemMainColumns">
+        <ul>
```
Comment (scott-confluent) [inline _includes/navMenus/courses.html:19]: not sure you can nest a 'ul' within a 'ul' directly

File: _includes/navMenus/courses.html [hunk 14]
```diff
@@ -0,0 +1,94 @@
+<ul class="menuGroup courses">
+
+    <ul class="wrapperSection">
```
Comment (scott-confluent) [inline _includes/navMenus/courses.html:3]: not sure you can nest a 'ul' within a 'ul' directly -- also remove empty line

File: _includes/navMenus/community.html [hunk 15]
```diff
@@ -0,0 +1,53 @@
+<ul class="menuGroup community">
+    <ul class="wrapperSection">
```
Comment (scott-confluent) [inline _includes/navMenus/community.html:2]: not sure you can nest a 'ul' within a 'ul' directly

File: _includes/navMenus/learn.html (hunk 9 above)
Comment (bpurkaystha) [reply _includes/navMenus/learn.html:13]: The book says not to use `ul` under `ul` directly, but works anyway ...?

File: _includes/navMenus/learn.html (hunk 8 above)
Comment (bpurkaystha) [reply _includes/navMenus/learn.html:52]: done.

File: _includes/navMenus/learn.html (hunk 9 above)
Comment (bpurkaystha) [reply _includes/navMenus/learn.html]: Updated.

File: _includes/navMenus/build.html [hunk 16]
```diff
@@ -0,0 +1,41 @@
...
+            <span class="subline">Build a client app, explore use cases, and build on our demos and resources</span>
+            <a href="https://developer.confluent.io/#designbuildrun" class="navItemGroupCta">
+                Start Building
+                <img src="/tutorials/assets/img/icon-arrow-right-white.svg" width="15" height="15"/>
+            </a>
+        </div>
+    </div>
+    <div class="canonicalPadding">
+        <li class="navItem">
+            <a href="https://developer.confluent.io/get-started/java/">
```
Comment (hartantothio) [inline _includes/navMenus/build.html:14]: _nit_ - Probably don't need full url. I think something like `/get-started/java/` would work

File: _includes/navMenus/community.html [hunk 17]
```diff
@@ -0,0 +1,53 @@
...
+                Apache Kafka®️, and its ecosystems
+            </span>
+            <a href="https://www.confluent.io/community" class="navItemGroupCta">
+                Learn More
+                <img src="/tutorials/assets/img/icon-arrow-right-white.svg" width="15" height="15"/>
+            </a>
+        </div>
+    </div>
+    <div class="canonicalPadding">
+        <li class="navItem">
```
Comment (hartantothio) [inline _includes/navMenus/community.html:16]: `li` shouldn't be a children of `div`

**Comment by hartantothio** [comment]
Missing images
image

CTA in nav
- The background doesn't match CD
- There is a white border
- Hover state is different from CD
image

Close button positioning is not correct
image

KT (left) vs CD (right)
- Logo has some left spacing on KT
- Close button positioning
- Button text is not the same color
- Search icon appears smaller (or maybe CD is much bigger)
- 
image

File: _includes/navMenus/build.html [hunk 18]
```diff
@@ -0,0 +1,41 @@
...
+        <div class="navItemGroupWrapper">
+            <div class="title">DESIGN. BUILD. RUN.</div>
+            <span class="subline">Build a client app, explore use cases, and build on our demos and resources</span>
+            <a href="https://developer.confluent.io/#designbuildrun" class="navItemGroupCta">
+                Start Building
+                <img src="/tutorials/assets/img/icon-arrow-right-white.svg" width="15" height="15"/>
+            </a>
+        </div>
+    </div>
+    <div class="canonicalPadding">
```
[the remark this answers is not in the document]
Comment (bpurkaystha) [reply _includes/navMenus/build.html:12]: Updated.

**Comment by bpurkaystha** [comment]
@hartantothio , I tried to address a number of your comments (not all). If we want to be that precise with KT & CD, then it'll take much time (may not be possible by today at all).

**Comment by bpurkaystha** [comment]
Or it'd be best to find a more better solution: can we plan (or discuss) to maintain only one repo for both CD & KT instead of two?

**Review by scott-confluent — APPROVED** [review]
Thanks for all your work on this!

**Review by bbejeck — APPROVED** [review]
piggy backing off @scott-confluent approval

**Comment by bbejeck** [comment]
This PR is HTML, CSS and images only the failures are unrelated - the site built and deployed

**Pull request merged**